### ISSUE TITLE
Sync rating tag with file

### ISSUE THREAD

**Issue by rshea0** [rshea0/tuneup-prime#67]
Support either writing Engine rating to file or vice versa.

**Comment by Houseman-HH** [comment]
I hope there is a choice between file tag and engine tag

For example, the engine does not write the rating in the file tag !!

**Comment by rshea0** [comment]
Yes, you will be able to chose if you want to sync the Engine rating to the file, or the file rating to Engine. I can also add an option to not overwrite the rating if one already exists.
